src/index.html

```diff
@@ -13,6 +13,131 @@
     />
```

Comment:
Link to the CSS file is incorrect. SCSS is a preprocessor and you need to link to the compiled CSS file, not the SCSS file. Make sure to compile your SCSS and link to the resulting CSS file.